Global/global.js

```
@@ -0,0 +1,62 @@
+	// Close mobile menu when clicking on a link
+	mobileNavLinks.forEach(link => {
+		link.addEventListener('click', function () {
+			mobileMenuToggle.classList.remove('active');
+			mobileNav.classList.remove('active');
+			document.body.style.overflow = '';
+		});
+	});
+
+	// Close mobile menu when clicking outside
+	document.addEventListener('click', function (event) {
+		if (mobileNav && mobileNav.classList.contains('active')) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)